inc/options/minify.php

```diff
@@ -136,7 +136,7 @@
 			<?php
 			Util_Ui::config_item( array(
 				'key' => 'minify.js.enable',
-				'label' => '<acronym title="JavaScript">JS</acronym> minify settings:',
+				'label' => __( '<acronym title="JavaScript">JS</acronym>', 'w3-total-cache' ),
```

Comment:
Do not place markup inside of a translation function.  Use `sprint` with `__()` as a parameter.  Also, include a `// translators: 1:` comment when doing so.

Reply:
fixed